spec/functions/assert_supported_pe_version_spec.rb

```diff
@@ -26,4 +26,18 @@
       is_expected.to run.with_params('2019.8.7').and_return({ 'supported' => true })
     end
   end
+
+  context 'unsafe versions' do
+    it 'accepts PE versions that are too old' do
+      is_expected.to run.with_params('2018.1.0', true).and_return({ 'supported' => true })
```

Comment:
Ah, shoot. Just realized that the intuitive result here is that the assert function does NOT raise (good so far), but the return value should actually be `{ 'supported' => false }`.